### ISSUE TITLE
can't properly build binary package for ubuntu xenial

### ISSUE THREAD

**Issue by chaocrator** [ahlstromcj/sequencer64#145]
unfortunately, i know very little about debian packaging system, so this report will certainly be lame, but i ran into the following issue with building a binary package for ubuntu 16.04 (xenial):

if i unpack the source tarball and run dpkg-buildpackage from within the source directory, the build always fails.

but if i first run "./bootstrap -er" and then dpkg-buildpackage, the build succeeds.

i spent a few hours trying to put the damn thing to debian/rules, but nothing worked for me.

this is not a big deal for a local build, but prevents from putting a package to a ppa, since the procedure is not completely automatic.

**Comment by ahlstromcj** [comment]
At work right now, but check out the debian/README file.  If it needs
updating to work with PPA, let me know what's needed and I will add the
steps.  Good luck, I am still working on packaging.

Chris

**Comment by chaocrator** [comment]
yes, the procedure described in debian/README is the same that worked for me -- first bootstrapping, then building with debian packaging tools.
